libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryClient.java

```diff
@@ -26,13 +30,20 @@
   private String userAgent = null;
   private final TelemetryClientSettings setting;
   private final Logger logger;
+  private Context context;
```

Comment:
https://github.com/mapbox/mapbox-events-android/pull/19#pullrequestreview-80533874
> `TelemetryClient` sounds like a good place to include that functionality

If we need to pass in a new dependency (`Context`), I'd rather leave it as it was and expose a method (`createFullUserAgent`) within `TelemetryUtils`. Sorry if I didn't make myself clear.